src/applications/facility-locator/components/MapboxToken.js

```diff
@@ -0,0 +1,2 @@
+export const mapboxToken =
```

Comment:
Could we move this file the utils folder in facility-locator? And lower case the file name (since its not a component)?